File: src/main/java/ar/edu/unq/spring/persistence/repository/jpa/PersonajeDAO.java
package ar.edu.unq.spring.persistence.repository.jpa;

import ar.edu.unq.spring.persistence.dto.PersonajeJPADTO;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface PersonajeDAO extends JpaRepository<PersonajeJPADTO, Long> {
}

File: src/main/kotlin/ar/edu/unq/spring/PersonajeDAO.kt
package ar.edu.unq.spring

import ar.edu.unq.spring.modelo.Personaje
import org.springframework.data.jpa.repository.JpaRepository

/**
 * Tiene la responsabilidad de guardar y recuperar personajes desde
 * el medio persistente
 */
interface PersonajeDAO : JpaRepository<Personaje, Long> {


}
